include/openssl/asn1.h.in

```diff
@@ -74,19 +74,19 @@ extern "C" {
 # define V_ASN1_UTF8STRING               12
 # define V_ASN1_SEQUENCE                 16
 # define V_ASN1_SET                      17
-# define V_ASN1_NUMERICSTRING            18 /**/
+# define V_ASN1_NUMERICSTRING            18
 # define V_ASN1_PRINTABLESTRING          19
 # define V_ASN1_T61STRING                20
 # define V_ASN1_TELETEXSTRING            20/* alias */
-# define V_ASN1_VIDEOTEXSTRING           21 /**/
+# define V_ASN1_VIDEOTEXSTRING           21
```

Comment:
As you're touching this - you could also add a space before `/* alias */` here and below